multidim-interop/impl/js/v0.45/Makefile

```diff
@@ -4,12 +4,16 @@ commitSha := 41641f1a7656aa654234f6f849b1749786867121
 # TODO Enable webkit once https://github.com/libp2p/js-libp2p/pull/1627 is in
 all: image.json chromium-image.json firefox-image.json
 
-chromium-image.json: image.json BrowserDockerfile
```

Comment:
The dependency of `BrowserDockerfile` is still valid, right? In other words, changing the `BrowserDockerfile` should lead to a rebuild of the `chromium-image.json` target.